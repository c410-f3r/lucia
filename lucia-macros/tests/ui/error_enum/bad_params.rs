#![allow(incomplete_features)]
#![feature(async_fn_in_trait, impl_trait_projections)]

#[lucia_macros::pkg(api(Foo), data_format(json))]
mod pkg {
  #[derive(Debug)]
  #[pkg::params]
  pub struct Rfewr;

  #[derive(Debug)]
  #[pkg::req_data]
  pub struct Req;

  #[pkg::res_data]
  struct Res;
}

fn main() {
}
