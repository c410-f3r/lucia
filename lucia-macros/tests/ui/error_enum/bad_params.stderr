error: Parameters must end with the `Params` suffix.
 --> tests/ui/error_enum/bad_params.rs:8:14
  |
8 |   pub struct Rfewr;
  |              ^^^^^
